code/obj/machinery/clone_rack.dm

```diff
@@ -39,7 +39,7 @@ TYPEINFO(/obj/machinery/disk_rack)
 /obj/machinery/disk_rack/ui_interact(mob/user, datum/tgui/ui)
 	ui = tgui_process.try_update_ui(user, src, ui)
 	if (!ui)
-		ui = new(user, src, "DiskRack")
+		ui = new(user, src, "DiskRack", src.name)
```

Comment:
oh I deliberately didn't do this because it'll show up labels which always feels weird, maybe use `initial(src.name)` instead?